webui/react/src/e2e/models/components/ModelRegistry.ts

```diff
@@ -1,8 +1,6 @@
-import { NamedComponent } from 'e2e/models/common/base/BaseComponent';
+import { BaseReactFragment } from 'e2e/models/common/base/BaseReactFragment';
 
 /**
  * Represents the ModelRegistry component in src/components/ModelRegistry.tsx
  */
-export class ModelRegistry extends NamedComponent {
-  defaultSelector: string = '[data-testid=modelRegistry]';
-}
+export class ModelRegistry extends BaseReactFragment {}
```

Comment:
https://github.com/determined-ai/determined/blob/main/webui/react/src/components/ModelRegistry.tsx#L675
model registry returns a react fragment, so this thing should be a react fragment